src/lib/assets/icons/Sort.tsx

```diff
@@ -20,7 +21,7 @@ const SortIcon: React.FunctionComponent<
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
-    {direction === Direction.DESC ? (
+    {(direction as string).toUpperCase() === Direction.DESC ? (
```

Comment:
Direction (used by Synapse API) is 'ASC' or 'DESC' and SortOrder is either 'asc' or 'desc' so toUpperCase does the trick